common/lib/capa/capa/templates/demandhint.html

```diff
@@ -0,0 +1,5 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
You need to set the page-level directive for escaping here. <%page expression_filter="h"/>.
Details-- http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#making-legacy-mako-templates-safe-by-default

Also please rebase on master so the safe template linter will run on your PR.